lib/Capabilities.php

```diff
@@ -46,8 +75,44 @@ public function getCapabilities() {
 					'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
 					'application/vnd.openxmlformats-officedocument.presentationml.presentation',
 				],
+				'collabora' => $collaboraCapabilities,
+				'direct_editing' => false, //TODO: fix once proper capability is available
+				'templates' => false, //TODO: fix once proper capability is available
```

Comment:
@tobiasKaminsky @marinofaggiana please check this value before you show the 'create new file' - collabora thingy. As it is not available by default.

Reply:
So you can have direct editing (cool webview) without templates?

Reply:
yes

Reply:
@rullzer can you update our test server, so we can test it?